packages/icons/src/custom-banner.svg

```diff
@@ -0,0 +1 @@
+<svg width="32" height="33" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.5 4.981a1 1 0 0 0-1 1v21a1 1 0 0 0 1 1h23a1 1 0 0 0 1-1v-21a1 1 0 0 0-1-1h-23Zm0 1h23v2.772h-23V5.98Zm0 3.772h23V26.98h-23V9.753Z" fill="#0278D5"/><path d="M27.5 5.981h-23v2.772h23V5.98Z" fill="#0278D5"/></svg>
```

Comment:
Don't use height / width. Use `viewBox`. Let's update this icon.
Further, replace the fill value with `currentColor` instead of `#0278D5`

General guidelines: https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices